reconcile/utils/vault.py

```diff
@@ -166,7 +181,8 @@ def _read_all_v2(self, path, version):
             raise SecretNotFound(path)
 
         data = secret["data"]["data"]
-        return data
+        secret_version = secret["data"]["metadata"]["version"]
+        return data, secret_version
```

Comment:
nit: since you have modified this function, please add type hints.

Reply:
Added type hints

Reply:
Hmmm, I think the was left in your computer, since I see no type hints in `_read_all_v2` 😅

Reply:
Submitted 😢